README.md

```diff
@@ -50,6 +50,9 @@ For instructions on installing the root server, see [Installing a New Root Serve
 - Added initial support for time zones, hidden behind feature flag.
 - Add a "publish" checkbox to the NAWS import functionality to allow new meetings to be either published or unpublished.
 - Fixed an issue where map searches using `geo_width` could return incomplete results when filtered by service body.
+- Fixed an issue with which meeting formats are selected for a NAWS export. NAWS exports are limited to 5 formats.
```

Comment:
I think you may have some unneeded newlines in this new comment. This should all really just go on one line.